Circuit/Circuit.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Is there any reason to change this? Because it'll make it hard/impossible to use those libraries in other projects, especially that we've been talking about publishing a CLI or library to use in other projects. If there is no specific API in net6.0-windows that we must have, then it's better to leave it as netstandard2.0 and let the consumer decide what target framework to use 😉

Reply:
OK, I will try another version of this. I'm confused as to why we needed multiple targets before though?

Reply:
I'm struggling with understanding when we should use netstandard2.0 and when we should use something like net6.0-windows. I'm trying to change `ComputerAlgebra` now (it seems like that would also need to change in addition to `Circuit` to have any benefit), but it seems like in order to build, all the projects need to target the same framework?

Reply:
I've been using net48 for debug purposes, as it allows for saving generated code to disk. Now I use some Nuget package that spills out a pseudo-code from expression tree, so we don't need it anymore. So targeting just net6.0 is fine in projects that were using multiple targets. Just leave the netstandard projects unchanged, as it won't hurt anyone, and will be useful in near future. 😁

Reply:
> I'm struggling with understanding when we should use netstandard2.0 and when we should use something like net6.0-windows. I'm trying to change `ComputerAlgebra` now (it seems like that would also need to change in addition to `Circuit` to have any benefit), but it seems like in order to build, all the projects need to target the same framework?

You can think of *netstandard2.0* as a kind of interface, that is implemented by various frameworks, like net48 or net6.0, so library targeting netstandard2.0 can be used by all those frameworks without recompiling. But there will be some framework-specific API's missing. But it's fine if you don't need them. 😉 You have to use a _concrete_ target framework when you are building an _.exe_ file, like a WPF or console app. Hope it helps 😉